script.js

```diff
@@ -1,15 +1,130 @@
+
+// VARIABLES
```

Comment:
Kindly remove this comment and also comments on line 15 and 31.

Reply:
Thank you. We'll do the changes